packages/foam-core/src/model/tags.ts

```diff
@@ -0,0 +1,183 @@
+import { FoamWorkspace, IDisposable, URI } from '../index';
+import { Resource } from './note';
+
+export type TagMetadata = { title: string; uri: URI };
```

Comment:
where is the title needed?

Reply:
Ah, I've kept that one in for the tag view explorer. But we are resolving the entire resource by URI at that point. We could stop the resolving, but then we lose the hover functionality, or remove the title from the tag object.

Reply:
For now, I'll remove it. Saves extra work in keeping things up to date as well.